ui/src/shared/components/RefreshingGraph.tsx

```diff
@@ -140,7 +165,7 @@ class RefreshingGraph extends PureComponent<Props> {
             case CellType.Gauge:
               return this.gauge(timeSeries)
             default:
-              return this.lineGraph(timeSeries, loading)
+              return this.lineGraph(timeSeries, timeSeriesFlux, loading)
```

Comment:
will it ever be the case that both `timeSeries` and `timeSeriesFlux` have values? if not, maybe two different `lineGraph` functions for the two different data source types would be more straightforward. as another developer reading this, i'm confused about two possible sets of data being passed into a single function, and what expectations i can have about what it will do.

Reply:
no because you will only ever be making either a flux query or an influxql query. rather than making another function, i can update some of the variable names to make it more clear. instead of `timeSeries` and `timeSeriesFlux` it can be `timeSeriesInfluxQL` and `timeSeriesFlux` so its clear why there are two datas.

Reply:
so then i don't think i would expect two different arguments where only one is populated. like, it could be a polymorphic function if nothing else, that maybe you pass an options param  in to control whether it's for influxql or flux for now, and others later

Reply:
i was mistaken, they will both have values. if its a flux query, then the influxql data will be an empty array and vice versa. so they will not be null values. The way line graph is knowing which way to parse the time series data is by whether the flux data or the influxql data is populated. however, i can move this parsing up a level to refreshing graph and depending on what type of time series data we have there, it can be parsed accordingly and then only one data will actually be passed into line graph.

Reply:
yeah. personally i would expect that `lineGraph` would expect only one set of data passed in, if it's only ever rendering one data set, and if need be it can be configurable for various data shapes at the time of invocation

Reply:
actually, that may not work since we because of the way we handle single stat + line graphs. it uses that unparsed data

Reply:
(and i would consider an empty array effectively an empty/nil value)

Reply:
The next story i am working on to update the way we are parsing flux to dygraph is going to change the way this will work significantly, I am going to put this off until that story so I dont have to rework it all twice. But overall, it makes sense to not have two sets of data being passed into line graph and ill make sure that makes it in.